### PR TITLE
Developement 3.0: Display warning rather than returning an error for non existent sources

### PR DESCRIPTION
**Pull request by cclerget**
**Description of the Pull Request (PR):**

This PR fix issue #1914

### CODE REVIEW

**Comment by GodloveD** [comment]
This seems like it is a fix for #1914.  Not 1904.  @cclerget can you confirm?

**Comment by cclerget** [comment]
@GodloveD Oops yes :). Fix that. Thanks Dave !

**Comment by jmstover** [comment]
A non-existent `/etc/resolv.conf` causes this to still error with a FATAL error

File: src/pkg/util/fs/layout/layer/overlay/overlay.go [hunk 1]
```diff
@@ -127,7 +128,8 @@ func (o *Overlay) createLayer(rootFsPath string, system *mount.System) error {
 			}
 			if point.Type == "" {
 				if err := syscall.Stat(point.Source, st); os.IsNotExist(err) {
-					return fmt.Errorf("stat failed for %s: %s", point.Source, err)
+					sylog.Warningf("skipping mount of %s: %s", point.Source, err)
+					continue
```
Comment (bauerm97) [inline src/pkg/util/fs/layout/layer/overlay/overlay.go]: Should we still check `} else if err != nil {` to error out if there's some other form of error that's not `os.IsNotExist()`?

**Review by GodloveD — DISMISSED** [review]
After @bauerm97 's suggested change I think this is good to go.

**Comment by cclerget** [comment]
@GodloveD @bauerm97 Finally changes to catch all errors like it's done with underlay

**Review by GodloveD — APPROVED** [review]
lgtm

**Pull request merged**